header_checks/meson.build

```diff
@@ -219,6 +219,9 @@ endif
 # libcheck
 check = dependency('check', fallback : ['check', 'check_dep'])
 
+# freetype2
+freetype2 = dependency('freetype2', fallback : ['freetype2', 'freetype_dep'])
```

Comment:
shouldn't it be `freetype2_dep`?

Reply:
If you're saying it should be `freetype2_dep = [...]`: nope, `freetype2` is the variable used around EFL (while `freetype2_dep` has no results). 

If you're saying it should be `['freetype2', 'freetype2_dep']`: also nope, [see the wrap's meson variable](https://github.com/mesonbuild/freetype2/blob/2.9.1/meson.build#L227).

Reply:
you're right  =)